mod-template-engine/src/main/resources/domain/mod-template-engine/features/templateEngine.feature

```diff
@@ -8,4 +8,64 @@ Feature: Template engine
   Scenario: Get all templates
     Given path 'templates'
     When method GET
-    Then status 200
\ No newline at end of file
+    Then status 200
+
```

Comment:
Please split these tests into separate feature-files by endpoint. Something like:
`templates.feature` for requests to `/templates`
`templateRequest.feature` for requests to `/template-request`

Take tests for `mod-feesfines` as an example.

Reply:
Fixed.